internal/cmd/cmd_expiretime.go

```diff
@@ -0,0 +1,43 @@
+package cmd
+
+import (
+	dstore "github.com/dicedb/dice/internal/store"
+	"github.com/dicedb/dicedb-go/wire"
+)
+var cExpireTime = &DiceDBCommand{
+	Name: "EXPIRETIME",
+	HelpShort: `EXPIRETIME returns the absolute Unix timestamp (since January 1, 1970) in seconds
+	at which the given key will expire`,
```

Comment:
Change this to `EXPIRETIME returns the absolute Unix timestamp in seconds at which the given key will expire`.

Reason: We expect the consumer of the database would know that Unix timestamp is. The date information is redundant.

Reply:
Will change the help short text.